src/shared/ui/Box/index.tsx

```diff
@@ -12,8 +12,8 @@ import MUIBox, { BoxProps } from '@mui/material/Box';
  * )
  */
 
-const Box = forwardRef<HTMLDivElement, BoxProps>((props: BoxProps) => {
-  return <MUIBox {...props} />;
+const Box = forwardRef<HTMLDivElement, BoxProps>((props, ref) => {
+  return <MUIBox {...props} ref={ref} />;
```

Comment:
This doesn't really make or break anything. But technically the proper way to use `forwardRef` is to always define a 2-parameters function, with `ref` being the 2nd parameter. The old way also _works_ because `ref` is part of `...props`, but my IDE gave me a warning about the proper usage format, so I just changed it.